src/store/state/state-model.ts

```diff
@@ -217,5 +217,33 @@ export const stateModel: StateModelIF = {
     shareStructure: false,
     incorporationAgreement: false
   },
-  summaryMode: false
+  summaryMode: false,
+  currentFees: {
```

Comment:
There is no reason to initialize this to zeros any more. I recommend just initializing this (and feePrices below) to null, since you now check for that when you "get" values from it. (Plus it makes things consistent.)

Reply:
👍

Reply:
Oh, what I actually meant was to use `currentFees: null` but I see you've gone the other way (which is better!).

---

So, here, please assign the empty IF, eg:
```
currentFees: cloneDeep(emptyFees),
feePrices: cloneDeep(emptyFees),
```

(You'll have to `import { cloneDeep } from 'lodash'`.)

(You have to clone the object otherwise boht variables point to the same instance of it.)